src/universal/components/TaskWatermark.js

```diff
@@ -1,56 +1,52 @@
-import {css} from 'aphrodite-local-styles/no-important';
 import React from 'react';
 import FontAwesome from 'react-fontawesome';
 import ui from 'universal/styles/ui';
-import withStyles from 'universal/styles/withStyles';
 import {GITHUB} from 'universal/utils/constants';
 import PropTypes from 'prop-types';
+import styled from 'react-emotion';
 
 const iconLookup = {
   [GITHUB]: 'github'
 };
 
+const WatermarkBlock = styled('div')({
```

Comment:
can you add a `pointerEvents: 'none'` to all watermarks? that way, if a link every appears under a watermark, it's still clickable. (we had that bug on the invoices)

Reply:
Sure thing! In this case the watermark is using `z-index` to be underneath all card content including the menu toggles in the footer, so it’s not an issue.